restapi.go

```diff
@@ -269,7 +251,7 @@ func (s *Session) Logout() (err error) {
 // userID    : A user ID or "@me" which is a shortcut of current user ID
 func (s *Session) User(userID string) (st *User, err error) {
 
-	body, err := s.Request("GET", EndpointUser(userID), nil)
+	body, err := s.RequestWithBucketID("GET", EndpointUser(userID), nil, EndpointUser(""))
```

Comment:
I updated some of them to this, did you mean to avoid the strings.SplitN call when using Request as well?